staff/daniel-villegas/cv-styled/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The CV</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet"> 
+	<link href="https://fonts.googleapis.com/css?family=Gamja+Flower" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <section class="section__title">
+            <h1>Curriculum</h1>
```

Comment:
Maybe you should use a header tag instead of a section

Reply:
Indeed!